easybuild/easyblocks/generic/configuremake.py

```diff
@@ -348,7 +350,12 @@ def test_step(self):
         """
 
         if self.cfg['runtest']:
```

Comment:
Hm, i just realized this leads the situation that you need to specify something in teh runtest part, else testing won't be considered. 
Perhaps change this to 
```python
if self.cfg['runtest'] or self.cfd['test_cmd'] != DEFAULT_TEST_CMD:
```

Reply:
@Micket Makes sense, please open a PR for that ASAP, maybe (just maybe) we can still include this into EasyBuild v4.5.5

Reply:
followed up in #2737